### PR TITLE
SFDX service deployment

### PR DESCRIPTION
**Pull request by magdielhf**
## Summary

Deploys selected services to scratch orgs, on its current version the script assumes the scratch org was already created.

The script is located inside the `install` folder and the configuration file is located on the `config/SFDXService` folder.

## Other Information

Usage: /sfdx-deploy-service.sh -c -s discovery -u test-yqejio8nyef5@company.net

### CODE REVIEW

File: install/sfdx-deploy-service.sh [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+echo "-u          Username associated with scratch org";
+echo "";
+echo "# Using the -c option to include the core files as part of this deployment";
+echo "# by default the core files are not included";
+echo "Example: $0 -c -s discovery -u myusername@organization.net"
+echo "";
+exit 1;
+fi
+
+<<<<<<< HEAD
```
Comment (germanattanasio) [inline install/sfdx-deploy-service.sh]: remove `<<<<<<< HEAD`

File: install/sfdx-deploy-service.sh [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+echo -e "executing script..."
+=======
+echo -e "running script..."
+>>>>>>> 5ecf64d6e90c2d4cc16adbda32b3d846b3f003cd
```
Comment (germanattanasio) [inline install/sfdx-deploy-service.sh]: remove `>>>>>>> 5ecf64d6e90c2d4cc16adbda32b3d846b3f003cd`

File: install/sfdx-deploy-service.sh (hunk 1 above)
Comment (magdielhf) [reply install/sfdx-deploy-service.sh]: fixed

File: install/sfdx-deploy-service.sh (hunk 2 above)
Comment (magdielhf) [reply install/sfdx-deploy-service.sh]: fixed

**Pull request merged**